### PR TITLE
Bao/PAF-1

### PR DESCRIPTION
**Pull request by baosphoa0403**
Set Up Layout And Set Routing Layout

### CODE REVIEW

File: src/pages/DashboardAdmin/Header/Header.tsx [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+    <>
+      <Box
+        sx={{
+          width: '100%',
+          display: 'flex',
+          alignItems: 'center',
+          [theme.breakpoints.down('md')]: {
+            width: 'auto',
+          },
+          // padding: '10px',
```
Comment (CNDvn) [inline src/pages/DashboardAdmin/Header/Header.tsx]: delete, ok

File: src/pages/DashboardAdmin/Header/NotificationSection/Notification.tsx [hunk 2]
```diff
@@ -0,0 +1,315 @@
...
+} from '@mui/material';
+
+// assets
+import {
+  IconBrandTelegram,
+  IconBuildingStore,
+  IconMailbox,
+  IconPhoto,
+} from '@tabler/icons';
+// import User1 from '../../../../../svg/user-round.svg';
```
Comment (CNDvn) [inline src/pages/DashboardAdmin/Header/NotificationSection/Notification.tsx:28]: same above

File: src/pages/DashboardAdmin/MainLayout/Dashboard/Dashboard.tsx [hunk 3]
```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+// material-ui
+import { Grid } from '@mui/material';
+import { gridSpacing } from 'constants/breakpoint';
+import { useTheme } from '@mui/material/styles';
+// import TableUser from './TableUser/tableUser';
```
Comment (CNDvn) [inline src/pages/DashboardAdmin/MainLayout/Dashboard/Dashboard.tsx]: same above

File: src/pages/DashboardAdmin/MainLayout/Dashboard/FormAddUser/formAddUser.tsx [hunk 4]
```diff
@@ -0,0 +1,228 @@
...
+              <OutlinedInput
+                className={classes.input}
+                id="outlined-adornment-email-login"
+                // onChange={(e: React.ChangeEvent<HTMLInputElement>): void => {
+                //   handleChange(e);
+                // }}
+                // onBlur={(e: FocusEvent<HTMLInputElement>): void => {
+                //   handleBlur(e);
+                // }}
+                // value={values.username}
```
Comment (CNDvn) [inline src/pages/DashboardAdmin/MainLayout/Dashboard/FormAddUser/formAddUser.tsx]: what is this ????

File: src/pages/DashboardAdmin/MainLayout/Dashboard/FormAddUser/formAddUser.tsx [hunk 5]
```diff
@@ -0,0 +1,228 @@
...
+                inputProps={{}}
+              />
+              {/* {touched.username && errors.username && (
+                <FormHelperText
+                  error
+                  id="standard-weight-helper-text-username-login"
+                >
+                  {errors.username}
+                </FormHelperText>
+              )} */}
```
Comment (CNDvn) [inline src/pages/DashboardAdmin/MainLayout/Dashboard/FormAddUser/formAddUser.tsx]: same above

File: src/pages/DashboardAdmin/MainLayout/Dashboard/FormAddUser/formAddUser.tsx [hunk 6]
```diff
@@ -0,0 +1,228 @@
...
+              <OutlinedInput
+                className={classes.input}
+                id="outlined-adornment-email-login"
+                // onChange={(e: React.ChangeEvent<HTMLInputElement>): void => {
+                //   handleChange(e);
+                // }}
+                // onBlur={(e: FocusEvent<HTMLInputElement>): void => {
+                //   handleBlur(e);
+                // }}
+                // value={values.username}
```
Comment (CNDvn) [inline src/pages/DashboardAdmin/MainLayout/Dashboard/FormAddUser/formAddUser.tsx]: same above

File: src/pages/DashboardAdmin/MainLayout/Dashboard/FormAddUser/formAddUser.tsx [hunk 7]
```diff
@@ -0,0 +1,228 @@
...
+                inputProps={{}}
+              />
+              {/* {touched.username && errors.username && (
+                <FormHelperText
+                  error
+                  id="standard-weight-helper-text-username-login"
+                >
+                  {errors.username}
+                </FormHelperText>
+              )} */}
```
Comment (CNDvn) [inline src/pages/DashboardAdmin/MainLayout/Dashboard/FormAddUser/formAddUser.tsx]: same above

File: src/pages/DashboardAdmin/MainLayout/Dashboard/FormAddUser/formAddUser.tsx [hunk 8]
```diff
@@ -0,0 +1,228 @@
...
+              <OutlinedInput
+                className={classes.input}
+                id="outlined-adornment-email-login"
+                // onChange={(e: React.ChangeEvent<HTMLInputElement>): void => {
+                //   handleChange(e);
+                // }}
+                // onBlur={(e: FocusEvent<HTMLInputElement>): void => {
+                //   handleBlur(e);
+                // }}
+                // value={values.username}
```
Comment (CNDvn) [inline src/pages/DashboardAdmin/MainLayout/Dashboard/FormAddUser/formAddUser.tsx]: SB

File: src/pages/DashboardAdmin/MainLayout/Dashboard/FormAddUser/formAddUser.tsx [hunk 9]
```diff
@@ -0,0 +1,228 @@
...
+                inputProps={{}}
+              />
+              {/* {touched.username && errors.username && (
+                <FormHelperText
+                  error
+                  id="standard-weight-helper-text-username-login"
+                >
+                  {errors.username}
+                </FormHelperText>
+              )} */}
```
Comment (CNDvn) [inline src/pages/DashboardAdmin/MainLayout/Dashboard/FormAddUser/formAddUser.tsx]: SB

File: src/pages/DashboardAdmin/MainLayout/Dashboard/FormAddUser/formAddUser.tsx [hunk 10]
```diff
@@ -0,0 +1,228 @@
...
+            </FormControl>
+            <FormControl
+              fullWidth
+              className={classes.formControl}
+              //   error={Boolean(touched.username && errors.username)}
```
Comment (CNDvn) [inline src/pages/DashboardAdmin/MainLayout/Dashboard/FormAddUser/formAddUser.tsx]: SB

File: src/pages/DashboardAdmin/MainLayout/Dashboard/FormAddUser/formAddUser.tsx [hunk 11]
```diff
@@ -0,0 +1,228 @@
...
+              <OutlinedInput
+                className={classes.input}
+                id="outlined-adornment-email-login"
+                // onChange={(e: React.ChangeEvent<HTMLInputElement>): void => {
+                //   handleChange(e);
+                // }}
+                // onBlur={(e: FocusEvent<HTMLInputElement>): void => {
+                //   handleBlur(e);
+                // }}
+                // value={values.username}
```
Comment (CNDvn) [inline src/pages/DashboardAdmin/MainLayout/Dashboard/FormAddUser/formAddUser.tsx]: SB

File: src/pages/DashboardAdmin/MainLayout/Dashboard/FormAddUser/formAddUser.tsx [hunk 12]
```diff
@@ -0,0 +1,228 @@
...
+                inputProps={{}}
+              />
+              {/* {touched.username && errors.username && (
+                <FormHelperText
+                  error
+                  id="standard-weight-helper-text-username-login"
+                >
+                  {errors.username}
+                </FormHelperText>
+              )} */}
```
Comment (CNDvn) [inline src/pages/DashboardAdmin/MainLayout/Dashboard/FormAddUser/formAddUser.tsx]: SB

File: src/pages/DashboardAdmin/MainLayout/Dashboard/FormAddUser/formAddUser.tsx [hunk 13]
```diff
@@ -0,0 +1,228 @@
...
+            </FormControl>
+            <FormControl
+              fullWidth
+              className={classes.formControl}
+              //   error={Boolean(touched.username && errors.username)}
```
Comment (CNDvn) [inline src/pages/DashboardAdmin/MainLayout/Dashboard/FormAddUser/formAddUser.tsx]: SB

File: src/pages/DashboardAdmin/MainLayout/Dashboard/FormAddUser/formAddUser.tsx [hunk 14]
```diff
@@ -0,0 +1,228 @@
...
+              <OutlinedInput
+                className={classes.input}
+                id="outlined-adornment-email-login"
+                // onChange={(e: React.ChangeEvent<HTMLInputElement>): void => {
+                //   handleChange(e);
+                // }}
+                // onBlur={(e: FocusEvent<HTMLInputElement>): void => {
+                //   handleBlur(e);
+                // }}
+                // value={values.username}
```
Comment (CNDvn) [inline src/pages/DashboardAdmin/MainLayout/Dashboard/FormAddUser/formAddUser.tsx]: SB

File: src/pages/DashboardAdmin/MainLayout/Dashboard/FormAddUser/formAddUser.tsx [hunk 15]
```diff
@@ -0,0 +1,228 @@
...
+                inputProps={{}}
+              />
+              {/* {touched.username && errors.username && (
+                <FormHelperText
+                  error
+                  id="standard-weight-helper-text-username-login"
+                >
+                  {errors.username}
+                </FormHelperText>
+              )} */}
```
Comment (CNDvn) [inline src/pages/DashboardAdmin/MainLayout/Dashboard/FormAddUser/formAddUser.tsx]: SB

File: src/pages/DashboardAdmin/MainLayout/Dashboard/TableUser/searchUser.tsx [hunk 16]
```diff
@@ -0,0 +1,40 @@
...
+  pointerEvents: 'none',
+  display: 'flex',
+  alignItems: 'center',
+  justifyContent: 'center',
+}));
+export const StyledInputBase = styled(InputBase)(({ theme }) => ({
+  color: 'inherit',
+  '& .MuiInputBase-input': {
+    padding: theme.spacing(1, 1, 1, 0),
+    // vertical padding + font size from searchIcon
```
Comment (CNDvn) [inline src/pages/DashboardAdmin/MainLayout/Dashboard/TableUser/searchUser.tsx]: SB

File: src/pages/DashboardAdmin/MainLayout/Dashboard/TableUser/tableUser.tsx [hunk 17]
```diff
@@ -0,0 +1,289 @@
...
+                        Delete
+                      </Button>
+                    </Box>
+                  </TableCell>
+                </TableRow>
+              );
+            })}
+        </TableBody>
+      </Table>
+      {/* chosen  RowsPerPage*/}
```
Comment (CNDvn) [inline src/pages/DashboardAdmin/MainLayout/Dashboard/TableUser/tableUser.tsx:237]: SB

File: src/pages/DashboardAdmin/MainLayout/index.tsx [hunk 18]
```diff
@@ -0,0 +1,47 @@
+import { AppBar, Box, CssBaseline, Toolbar } from '@mui/material';
+import { useTheme } from '@mui/material/styles';
+import React, { useState } from 'react';
+import Header from '../Header/Header';
+import Sidebar from '../SlideBar/SlideBar';
+import Dashboard from './Dashboard/Dashboard';
+// import TableUser from './Dashboard/TableUser/tableUser';
```
Comment (CNDvn) [inline src/pages/DashboardAdmin/MainLayout/index.tsx]: SB

**Review by CNDvn — CHANGES_REQUESTED** [review]
fix all, ok

File: src/pages/DashboardAdmin/MainLayout/Dashboard/TableUser/tableUser.tsx (hunk 17 above)
Comment (baosphoa0403) [reply src/pages/DashboardAdmin/MainLayout/Dashboard/TableUser/tableUser.tsx:237]: This is my comment divide code

**Review by CNDvn — APPROVED** [review]

**Review by lpduy2k — APPROVED** [review]

**Pull request merged**